src/main/java/com/exasol/dbbuilder/AbstractDatabaseObject.java

```diff
@@ -8,16 +8,19 @@
 public abstract class AbstractDatabaseObject implements DatabaseObject {
     protected DatabaseObjectWriter writer;
     protected String name;
+    protected final boolean attached;
```

Comment:
This name is not really clear to me. But I can't come up with any other variant :(

Reply:
Maybe invert the logic and call it `owned`? Or `ownedByDatabaseBuilder`?

Reply:
Maybe `attachedToDatabaseBuilder` ?

Reply:
Renamed to "owned" and improved JavaDoc in class `AbstractDatabaseObject`.